pkg/apb/client.go

```diff
@@ -185,16 +171,13 @@ func (c *Client) PullImage(imageName string) error {
 	return nil
 }
 
-func (c *Client) refreshLoginToken(clusterConfig ClusterConfig) error {
-	c.log.Debug("Refreshing login token...")
-	c.log.Debug("target: [ %s ]", clusterConfig.Target)
-	c.log.Debug("user: [ %s ]", clusterConfig.User)
-	c.log.Debug("password:[ %s ]", clusterConfig.Password)
+func (c *Client) Login() error {
+	c.log.Debug("Logging in to openshift...")
 
 	output, err := runCommand(
-		"oc", "login", "--insecure-skip-tls-verify", clusterConfig.Target,
-		"-u", clusterConfig.User,
-		"-p", clusterConfig.Password,
+    "oc", "login", "https://kubernetes.default",
+    "--certificate-authority", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+    "--token", "$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
```

Comment:
@fabianvf You can check if we're running locally or in cluster here:
https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/client.go#L78
Since every pod in the cluster is given a configfile, we'll check if that exists and use it. If it doesn't, then we use the local cluster config. That should give you the check you need.

After the TP we should really split out these code  paths to make this cleaner.